diff --git a/Setup/InstallData.php b/Setup/InstallData.php
index 9b2139d..0289fee 100644
--- a/Setup/InstallData.php
+++ b/Setup/InstallData.php
@@ -23,6 +23,20 @@ class InstallData implements InstallDataInterface
     private $categorySetupFactory;
 
     /**
+     * Root category ID
+     *
+     * @var int
+     */
+    private $rootCategoryId = 1;
+
+    /**
+     * Default category ID
+     *
+     * @var int
+     */
+    CONST DEFAULT_CATEGORY_ID = 2;
+
+    /**
      * Init
      *
      * @param CategorySetupFactory $categorySetupFactory
@@ -46,10 +60,10 @@ class InstallData implements InstallDataInterface
         $categorySetup->installEntities();
         // Create Root Catalog Node
         $categorySetup->createCategory()
-            ->load(1)
-            ->setId(1)
+            ->load($this->rootCategoryId)
+            ->setId($this->rootCategoryId)
             ->setStoreId(0)
-            ->setPath(1)
+            ->setPath($this->rootCategoryId)
             ->setLevel(0)
             ->setPosition(0)
             ->setChildrenCount(0)
@@ -59,10 +73,10 @@ class InstallData implements InstallDataInterface
 
         // Create Default Catalog Node
         $category = $categorySetup->createCategory();
-        $category->load(2)
-            ->setId(2)
+        $category->load(self::DEFAULT_CATEGORY_ID)
+            ->setId(self::DEFAULT_CATEGORY_ID)
             ->setStoreId(0)
-            ->setPath('1/2')
+            ->setPath($this->rootCategoryId . '/' . self::DEFAULT_CATEGORY_ID)
             ->setName('Default Category')
             ->setDisplayMode('PRODUCTS')
             ->setIsActive(1)
