diff --git a/Setup/InstallSchema.php b/Setup/InstallSchema.php
index 241028a..5f0031e 100644
--- a/Setup/InstallSchema.php
+++ b/Setup/InstallSchema.php
@@ -260,7 +260,7 @@ class InstallSchema implements InstallSchemaInterface
                 'group_id' => 1,
                 'website_id' => 1,
                 'name' => 'Main Website Store',
-                'root_category_id' => 2,
+                'root_category_id' => InstallData::DEFAULT_CATEGORY_ID,
                 'default_store_id' => 1
             ]
         );
